pandas/io/formats/css.py

```diff
@@ -27,12 +27,47 @@ def expand(self, prop, value: str):
     return expand
 
 
+def _border_expander(side: str = ""):
+    if side != "":
+        side = f"-{side}"
+
+    def expand(self, prop, value: str):
+        tokens = value.split()
+        if len(tokens) == 0 or len(tokens) > 3:
+            raise ValueError(f'Too many tokens provided to "{prop}" (expected 1-3)')
```

Comment:
for test coverage can you test this error is raised

Reply:
Do you think a warning here would be more appropriate than an Exception?

Reply:
generally speaking  bad CSS usually produces no warning in browsers and simply doesnt render. We dont have any warnings or errors elsewhere that try to detect bad-css. I might even be tempted to silently ignore?

Reply:
Thanks. I'll make it a warning. I want to keep something in there since `border: <width> <style> <color>` differs from the `margin: <top> <right> <bottom> <left>` shorthand